api/odilo.py

```diff
@@ -490,44 +971,150 @@ def get_url(self, limit, modification_date, offset):
 
         return url
 
+class MockOdiloAPI(OdiloAPI):
+    def patron_request(self, patron, pin, *args, **kwargs):
+        response = self._make_request(*args, **kwargs)
 
-class FullOdiloCollectionMonitor(OdiloCirculationMonitor):
-    """Monitor every single book in the Odilo collection.
+        # Modify the record of the request to include the patron information.
+        original_data = self.requests[-1]
 
-    This tells us about books added to the Odilo collection that
-    are not found in our collection.
+        # The last item in the record of the request is keyword arguments.
+        # Stick this information in there to minimize confusion.
+        original_data[-1]['_patron'] = patron
+        original_data[-1]['_pin'] = patron
```

Comment:
Should this be set to `pin` instead of `patron`?

Reply:
It was already set to "patron" in the existing code, but yes, I think it should be set to "pin" instead; I just changed it.